e2e/action-row.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { test, expect, type Page, type Locator } from '@playwright/test';
+
+const challengeButtons = [
+  'Instructions',
+  'index.html',
+  'styles.css',
+  'Console'
+];
+
+const editorButtons = ['index.html', 'styles.css'];
+
+let page: Page;
+let actionRow: Locator;
+let tabsRow: Locator;
```

Comment:
I'm not sure these should be globals?

Reply:
I left those variables as global to avoid repeating them in some of the tests, but I can remove them and convert them to locals or make them consts, though I'm not sure which option is better

Reply:
I generally advise against global variables. They can lead to unexpected behaviours. IMHO, better to query the page in each test.

Reply:
I removed the global vars and replaced them with querying functions